src/Components/Body.jsx

```diff
@@ -1,9 +1,35 @@
+import { Route, Routes } from "react-router-dom";
 import Reviews from "./Reviews";
+import ReviewSingle from "./ReviewSingle";
+import { useState, useEffect } from "react";
+import { getReviews, getReviewID, getCategories } from "../api";
+import { useParams } from "react-router-dom";
+
+export default function Body() {
+  const [reviews, setReviews] = useState([]);
+  // const { item_id } = useParams();
```

Comment:
if these comments are going to be needed for a later ticket putting them on a separate branch would be best practice